src/__tests__/cli-util.test.ts

```diff
@@ -1,6 +1,7 @@
+import { test } from 'vitest'
 import { parseArguments } from '../cli-util'
 
-test('parseArguments', () => {
+test('parseArguments', ({ expect }) => {
```

Comment:
`expect` can be imported from `vitest` too, is there any difference in doing this vs using the import?

Reply:
It can run asynchronously, so Vitest can parallelize the tests.
If it's imported globally, Vitest won't know which test is calling expect.